snowplow/iglu-client-embedded/schemas/com.metabase/instance_stats/jsonschema/1-0-1

```diff
@@ -0,0 +1,160 @@
+      }
+    },
+    "grouped_metrics": {
+      "description": "Grouped metrics",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "description": "The unique name of the grouped metric",
+            "type": "string",
+            "maxLength": 255
+          },
+          "values": {
+            "description": "Values for the grouped metric",
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
+                "group": {
+                  "description": "The group name",
+                  "type": "string",
+                  "maxLength": 255
+                },
+                "value": {
+                  "description": "The value for the group",
+                  "type": "number"
+                }
+              },
+              "required": ["group", "value"],
+              "additionalProperties": false
+            }
+          },
+          "tags": {
+            "description": "Tags that can be used flagging teams / features the metric belongs to",
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          }
+        },
+        "required": ["name", "values"],
+        "additionalProperties": false
+      }
+    },
+    "features": {
+      "description": "Features",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "description": "A single instance feature",
+        "properties": {
+          "name": {
+            "description": "The unique name of the feature",
+            "type": "string",
+            "maxLength": 255
+          },
+          "available": {
+            "description": "Whether the feature is available, i.e. can it be enabled/disabled or is it always on",
+            "type": "boolean"
+          },
+          "enabled": {
+            "description": "Whether the feature is enabled, i.e. can it be used by the users/instance",
+            "type": "boolean"
+          }
+        },
+        "required": ["name", "available", "enabled"],
+        "additionalProperties": true
+      }
+    },
+    "metadata": {
+      "description": "Metadata about the anonymous stats collection",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "description": "A single metadata key/value",
+        "properties": {
+          "key": {
+            "description": "The key for this metadata",
+            "type": "string",
+            "maxLength": 255
+          },
+          "value": {
+            "description": "The value of this metadata",
+            "type": ["string", "boolean", "integer", "null"],
+            "maxLength": 255,
+            "minimum": 0,
+            "maximum": 2147483647
+          }
+        },
+        "required": ["key", "value"]
+      }
+    }
+  },
+  "additionalProperties": false,
+  "required": ["instance_attributes", "features", "metadata"]
```

Comment:
Should we change this as well to make the new fields required?

Reply:
Well, we are adding the _schema_ for grouped metrics here, but not the data for it. so I will make "metrics" required. but we should hold off on "grouped_metrics" (which will be addable w/o updating the schema in a followup).

Reply:
Yep, saw it in the [other discussion](https://github.com/metabase/metabase/pull/48476#discussion_r1795761878) and commented there. 
The approach is fine for me as long as we don't run into issues such that we need to create a major version update for our schema (that always requires more work since it creates a new table in data land). I should have thought about this earlier when we created the first initial version of the schema. Let's double-check with Joao from SnowcatCloud.

Reply:
I believe we should not include grouped metrics until they're being created. We will still have to update the schema (to make them required, and change the type from `[object, null]`, to `object`. I have a copy of the grouped_metrics json schema handy and I can paste it into an issue for adding grouped_metrics.